### DISCUSSION TITLE
Really impressed with Seal

### DISCUSSION THREAD

**Discussion by OneEyedTiamat** [JunkFood02/Seal#1222] [General]
Hi,

I used Seal for the first time this week and I'm really impressed. Easy to use, convenient settings, can download unlisted playlists, **metadata**.

The only improvement I can think of (queue/schedule downloads) is being worked on.

I'm a bit concerned about being unable to remove my payment information, but I might just have to sponsor this one.

**Comment by JunkFood02** [comment]
Due to certain restrictions from the bank, I can only accept donations on Github. About the payment information issue, it seems that someone has already [complained about this](https://github.com/orgs/community/discussions/51727). Nevertheless, I'm honored that you like it! 🥰
